### ISSUE TITLE
NiAlphaProperty detection for Textures

### ISSUE THREAD

**Issue by neomonkeus** [niftools/blender_niftools_addon#25]
We need to add support to detect alpha channels in textures. The problem is that some texture use the alpha channel for other purposes.

**Comment by HENDRIX-ZT2** [comment]
When does the nif format check for an alpha channel in textures? Either it's there in a texture or it isn't, but the properties don't mind.

**Comment by HENDRIX-ZT2** [comment]
IMO this is a non-issue 🤷‍♂️

**Comment by neomonkeus** [comment]
Its more the decision whether or not to attach an NiAlphaProperty to the material on export.

**Comment by HENDRIX-ZT2** [comment]
Right, I would base that solely on the material's alpha blend mode. Only attach an alpha prop if b_mat is not 'OPAQUE'.

**Comment by neomonkeus** [comment]
If that value can be used to decide to add a NiAlphaProp, then that would be great. 
Just needs documenting at that point.
